python/ambassador_diag/diagd.py

```diff
@@ -1099,7 +1100,7 @@ def _load_ir(self, rqueue: queue.Queue, aconf: Config, fetcher: ResourceFetcher,
                 kind, namespace, update = app.ir.k8s_status_updates[name]
                 text = json.dumps(update)
 
-                self.logger.info(f"K8s status update: {kind} {resource_name}.{namespace}, {text}...")
+                # self.logger.info(f"K8s status update: {kind} {resource_name}.{namespace}, {text}...")
```

Comment:
Do we want to keep at least 1 line of log?

Reply:
There's still "KubeStatus UPDATE <pid>" and "KubeStatus DONE"

Reply:
We have output when the kubestatus command actually runs, still.